tests/helpers/ember.js

```diff
@@ -2,14 +2,38 @@
 
 var MockUI        = require('./mock-ui');
 var MockAnalytics = require('./mock-analytics');
-var Cli           = require('../../lib/cli');
+var cli           = require('../../lib/cli');
 
+/*
+  Accepts a single array argument, that contains the same
+  type of array as is available from a command line.
```

Comment:
might be worth mentioning that it is basically part of `process.argv`

process.argv is basically `[pathToNode, pathToFile, ...argsEmber-CliCares about]`

So

```
ember new foo
```

becomes `process.argv === ["path/to/node", "path/to/bin/ember", "new", "foo"]`